src/Fable.Promise.fsproj

```diff
@@ -14,6 +14,6 @@
     <Content Include="*.fsproj; *.fs" PackagePath="fable\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fable.Core" Version="3.1.5" />
+    <PackageReference Include="Fable.Core" Version="3.2.3" />
```

Comment:
Same comment as in https://github.com/fable-compiler/fable-fetch/pull/15#discussion_r559281122